src/Grids/inactive_node.jl

```diff
@@ -161,3 +161,9 @@ Return `true` when the location `(LX, LY, LZ)` lies on a boundary.
 """
 @inline boundary_node(i, j, k, grid, LX, LY, LZ) = peripheral_node(i, j, k, grid, LX, LY, LZ) & !inactive_node(i, j, k, grid, LX, LY, LZ)
 
+"""
+    bottommost_active_node(i, j, k, grid, LX, LY, LZ)
+
+Return `true` when the location `(LX, LY, LZ)` is the active cell closest to the bottom.
+"""
+@inline bottommost_active_node(i, j, k, grid, LX, LY, LZ) = active_node(i, j, k, grid, LX, LY, LZ) && inactive_node(i, j, k - 1, grid, LX, LY, LZ)
```

Comment:
```suggestion
@inline bottommost_active_node(i, j, k, grid, LX, LY, LZ) = active_node(i, j, k, grid, LX, LY, LZ) & inactive_node(i, j, k - 1, grid, LX, LY, LZ)
```

are the locations instantiated? If so they need to use instantiated notation (lowercase)

Reply:
I'm following the nomenclature for the rest of the file. I think every function there has an upper-case notation but receives instantiated locations. Do you want me to change them all?

Reply:
no, you can leave it. I didn't realize the problem is widespread